pkg/skaffold/build/local/local.go

```diff
@@ -57,6 +57,21 @@ func (b *Builder) PostBuild(ctx context.Context, _ io.Writer) error {
 	return nil
 }
 
+// Prune uses the docker API client to remove all images built with Skaffold
+func (b *Builder) Prune(ctx context.Context, _ io.Writer) error {
```

Comment:
I noticed that running `skaffold dev --cache-artifacts=false --no-prune=false` triggers a prune logic that calls this function, that goes through all the images built by Skaffold and prune them, due to the new prune logic in line 107, it doesn't find the images (were already deleted) and now it shows a warning message doing the clean up. Do you think that message is something we should avoid to print now?

Reply:
ahhhhh! Great catch! updated! Now we use localpruner to do the cleanup job which should track deleted images, so when the devloop finishes, we won't see the warnings.